deb-get

```diff
@@ -633,15 +613,15 @@ function deb_quickemu() {
     SUMMARY="Quickly create and run optimised Windows, macOS and Linux desktop virtual machines."
 }
 
-function deb_virtualbox-6.1() {
```

Comment:
Since the name of the package for VirtualBox was changed, it is not that simple to upgrade it. Please remove the changes made to it, so we can accept the Cider package.